.github/workflows/rust.yml

```diff
@@ -171,3 +177,29 @@ jobs:
         with:
           command: clippy
           args: --all-targets -- -D warnings
+
+  no_std:
+    name: Check no_std build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: aarch64-unknown-none
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v1.3.0
+
+      - name: Check no_std build
+        uses: actions-rs/cargo@v1.0.3
+        with:
+          command: check
+          # The aarch64-unknown-none doesn't support `std`, so this
+          # will fail if the crate is not fully no_std compatible.
+          args: --target aarch64-unknown-none --no-default-features --features serde,from-string,parser-ss58
```

Comment:
super nit: A new line here?

Reply:
YAML scares me too much; I wonder whether I can just insert a couple of newlines and it'll concat onto one line though (which iirc is the behaviour...)